call-home/src/main.rs

```diff
@@ -50,7 +50,7 @@ async fn main() {
 
 async fn run() -> anyhow::Result<()> {
     let args = CliArgs::args();
-    let version = RELEASE_VERSION.to_string();
+    let version = release_version();
```

Comment:
missing change to the clap to set the correct version, see [here](https://github.com/openebs/mayastor-extensions/blob/efe832075f7f2fd770a0a58286f8cd8fbf212403/k8s/plugin/src/main.rs#L20) as example

Reply:
Added the clap attributes.